templates/rc.local.erb

```diff
@@ -1,4 +1,6 @@
+<% if @operatingsystem != 'OpenBSD' %>
```

Comment:
Please add a dahs here to avoide newlines in the template:
-%>
instead of 
%>